### ISSUE TITLE
Responding Vendors

### ISSUE THREAD

**Issue by Resource-Data** [AlaskaDHSS/RFP-ARIES-AutomatedRenewals#94]

**Comment by nfaulknerAK** [comment]
Yes. All suggestions/questions as well as the answers will be publicly visible on the draft RFP. Additionally the proposals submitted for the final RFP are public knowledge and will be posted along with the Notice of Intent to Award.
